data/processed/hesa/fte_research_students.csv

```diff
@@ -0,0 +1,121 @@
+year,nuts_id,nuts_year_spec,fte_research_students
+2015/16,UKH2,2018,1305.0
+2016/17,UKH2,2018,1280.0
+2017/18,UKH2,2018,1270.0
```

Comment:
I see that these are all floats ending with `.0`, marking this to reference in the `precision`/`format` discussion

Reply:
Ok - these are now outputted as integers so there should be no issue.